rust/src/storage/file/mod.rs

```diff
@@ -31,6 +31,7 @@ mod rename;
 /// * Support for other platforms are not implemented at the moment.
 #[derive(Default, Debug)]
 pub struct FileStorageBackend {
+    #[allow(dead_code)]
     root: String,
```

Comment:
I think we can just remove this, see the issue you filed ;) https://github.com/delta-io/delta-rs/issues/488

Reply:
Oh, I saw someone wants to take that. :)